### DISCUSSION TITLE
Question on documentation

### DISCUSSION THREAD

**Discussion by enikidis** [mir-group/allegro#46] [Q&A]
### Hello ! I managed succesfully to install allegro and I am currently running a job on a gpu. Is there any place I can get a general idea on the output and what they actually mean.?
Train      #    Epoch      wal       LR       loss_f       loss_e         loss        f_mae       f_rmse        e_mae       e_rmse
! Train              24 32637.932    0.001      0.00142      0.00137       0.0156       0.0319       0.0596        0.752         1.26
! Validation         24 32637.932    0.001      0.00128      0.00114       0.0139       0.0283       0.0583        0.695          1.2
Wall time: 32637.932265501004
! Best model       24    0.014

As I understand this 0.014 is important and needs to decrease with every best model found. 
How should someone start and understanding this and not use it as a "blackbox" approach?

**Comment by Linux-cpp-lisp** [comment]
Best model is reporting whichever metric was set as `metrics_key`; from `full.yaml` where all trainer options are documented:
```
metrics_key: validation_loss                                                       # metrics used for scheduling and saving best model. Options: `set`_`quantity`, set can be either "train" or "validation, "quantity" can be loss or anything that appears in the validation batch step header, such as f_mae, f_rmse, e_mae, e_rmse
```
Default is `validation_loss`.
